src/contractTest/resources/application.yaml

```diff
@@ -233,15 +233,34 @@ document:
       C43A_draft_filename: C43ADraftCreateOrder.pdf
       C43A_final_template: PRL-ORDER-C43A-FINAL.docx
       C43A_final_filename: C43AFinalCreateOrder.pdf
+      prl_c49_draft_template: PRL-ORDER-C49-DRAFT-COMMON.docx
+      prl_c49_draft_filename: C49ADraftCreateOrder.pdf
```

Comment:
Please check if c49 implementation is still required? as I see https://tools.hmcts.net/jira/browse/PRL-1368 is withdrawn

Reply:
@cshiwarale  as discussed we disabled this option from UI.